### ISSUE TITLE
Sections not supported in block-level table cells with "a" operator

### ISSUE THREAD

**Issue by owenh000** [owenh000/asciidoctor-multipage#18]
Unfortunately it is possible to put a section header inside a table cell with the "a" operator. The asciidoctor-multipage extension may not handle these sections as expected.

I hope section headers are seldom, if ever, being used inside table cells. Thus, fixing this issue is ultra-low priority. If however you want to see this fixed then this is the issue for you to comment on and/or work on.

**Comment by tajmone** [comment]
> I hope section headers are seldom, if ever, being used inside table cells. 

Indeed, and even if they are they'll most likely be _[discrete headings]_. 

> The asciidoctor-multipage extension may not handle these sections as expected.

Any chance that the `discrete` attribute could be checked to skip such headings and prevent breakage? (this should apply to any block, since discrete headings are allows in any block, usually to mimic the style of section headings).

> Thus, fixing this issue is ultra-low priority.

Definitely not urgent, but still worth sharing thoughts on this, for the solution might be simple (as proposed above).



[discrete headings]: https://docs.asciidoctor.org/asciidoc/latest/blocks/discrete-headings/

**Comment by owenh000** [comment]
@tajmone, good idea; thanks for reminder about discrete headings.
